### PR TITLE
Add test coverage and badges to docs

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?
⭐️ Test coverage with coveralls

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # VTEX Header
 
+[![Build Status](https://travis-ci.org/vtex-apps/store-header.svg?branch=master)](https://travis-ci.org/vtex-apps/store-header)
```
Comment (GustavoMA) [inline README.md]: There is already a Build Status badge under the `Travis CI` section.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+[![Coverage Status](https://coveralls.io/repos/github/vtex-apps/store-header/badge.svg?branch=master)](https://coveralls.io/github/vtex-apps/store-header?branch=master)
```
Comment (GustavoMA) [inline README.md]: Move to the `Travis CI` section.

File: .travis.yml [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
 cache:
   directories:
     - 'node_modules'
+
 node_js:
   - '10'
+
 before_install:
   - cd react
+  - yarn add coveralls
```
Comment (GustavoMA) [inline .travis.yml]: Why this isn't in the dev dependencies in the `package.json`?

Comment (matheusps) [reply .travis.yml]: This dependency is only useful on CI process

**Pull request merged**